crates/typst/src/layout/page.rs

```diff
@@ -228,6 +228,10 @@ pub struct PageElem {
 
     /// The page's header. Fills the top margin of each page.
     ///
+    /// - Content: Shows the content as the header.
+    /// - `{auto}`: Shows the page number if `number-align` is set to `top`.
+    /// - `none`: Suppresses the header.
```

Comment:
```suggestion
    /// - `{none}`: Suppresses the header.
```
Same for the footer.